packages/api/src/hooks/useBalance.tsx

```diff
@@ -0,0 +1,28 @@
+import { useFetch } from '@deriv/api';
+import { useMemo } from 'react';
+
+/** A custom hook that gets the balance for user accounts.
+ * If no account argument is provided, by default the hook will return the balance of all accounts.
+ *
+ * @param account {string} - If set to 'all', return the balances of all accounts one by one.
+ *                           If set to 'current', return the balance of current account.
+ *                           If set as an account id, return the balance of that account.
+ *                           Default is set to 'all'.
+ */
+const useBalance = (account?: string) => {
```

Comment:
Made `account` payload as an argument to make `useBalance` more generic, previous code:
<img width="588" alt="Screenshot 2023-08-21 at 5 20 54 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/0427592c-897f-42f1-8ecf-7649ada26012">